pkg/controller/attach_detach.go

```diff
@@ -27,7 +28,7 @@ import (
 	"google.golang.org/grpc/status"
 	v1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
If you plan to change the package import name, let's stick with the same naming as in other files.
https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/c6813a5c47afefcaa13df68f9f93bc37911b4ebf/pkg/provisioner/crdprovisioner.go#L32